oaknational/Oak-Web-Application
# Feat/lesq 153/html lists a11y [LESQ-153]

## Description

Music year: 2020

- Created `GridList` and `GridAreaListItem` components to make semantic lists in grids possible
- Refactored the `LessonElementLinks` on the plan a lesson page to make it easier to work with in a list
- Refactored the bio modal layout so it's all in a single list instead of nested
- Wrapped other elements in `ul`, `ol` and `li` where appropriate

## How to test

Use the inspector to view the elements on the page

1. Go to https://deploy-preview-2146--oak-web-application.netlify.thenational.academy/lesson-planning
2. You should see the icons at the top of the page are in a list, as well as the numbered list below (ACs 1 & 2)
4. Go to https://deploy-preview-2146--oak-web-application.netlify.thenational.academy/about-us/leadership and https://deploy-preview-2146--oak-web-application.netlify.thenational.academy/about-us/board
5. You should see the links in the header are in a list (AC 3) and the bio cards below are in a list (AC 4) as well as the documents on the /board page (AC 5)
6. Go to https://deploy-preview-2146--oak-web-application.netlify.thenational.academy/)about-us/partners
7. You should see the logos are in a list (AC 6)
8. Go to https://deploy-preview-2146--oak-web-application.netlify.thenational.academy/teachers/key-stages/ks1/subjects
9. You should see the subject cards are in a list (AC 7)

It should otherwise look the same

## ACs
1. Plan a lesson link section (the circle link icons to different lesson components) should be a list of links
2.  Plan a lesson, the “1. Find the right lesson, 2. Tailor…” looks as though it should be a list, but in HTML it is a bunch of divs
3.  About us link section (who we are, leadership etc) should be a list of links
4.  List of board/leadership cards in about us should be lists
5.  Documents on the about-us/board page should be a list
6.  List of partner logos on about-us/partners should be a list
7.  List of subjects on a subject listing page
